qark/modules/filePermissions.py

```diff
@@ -47,9 +47,9 @@ def start(queue, height):
     file_ww = r'MODE_WORLD_WRITEABLE'
     for i in text_scan(common.java_files, file_ww):
         if len(i) > 0:
-            report.write(IssueType.FileSystem, IssueSeverity.High,
-                         common.config.get('qarkhelper', 'WW_FILE') + str(i[0])
-                         + "<br>" + str(i[1]))
+            #report.write(IssueType.FileSystem, IssueSeverity.High,
+            #             common.config.get('qarkhelper', 'WW_FILE') + str(i[0])
+            #             + "<br>" + str(i[1]))
```

Comment:
Commented this out, it is redundant with lines 53-63 below and also throws errors when compiling the report.